css/properties/alignment-baseline.json

```diff
@@ -27,7 +27,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤4"
```

Comment:
I used [this test](http://bcd-examples.gh.caugner.de/css/alignment-baseline/) in Safari 5.0, and it doesn't appear to be supported yet:
<img width="1352" alt="image" src="https://github.com/user-attachments/assets/126beeab-caaf-4d81-9ac1-275f3a4992a5" />

However, it is supported in Safari 5.1:
<img width="1352" alt="image" src="https://github.com/user-attachments/assets/fc0f1cb1-9219-406a-92c2-94c7881b0870" />

So we can probably set all to "5.1".